# Array of person positions
std_msgs/Header header
PoseWithCov[] poses


